MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -21,7 +21,7 @@ class MapboxCoreNavigationTests: XCTestCase {
         navigation.resume()
         let depart = CLLocation(coordinate: CLLocationCoordinate2D(latitude: 37.795042, longitude: -122.413165), altitude: 1, horizontalAccuracy: 1, verticalAccuracy: 1, course: 0, speed: 10, timestamp: Date())
         
-        self.expectation(forNotification: RouteControllerDidPassSpokenInstructionPoint.rawValue, object: navigation) { (notification) -> Bool in
+        self.expectation(forNotification: NSNotification.Name(rawValue: RouteControllerDidPassSpokenInstructionPoint.rawValue), object: navigation) { (notification) -> Bool in
```

Comment:
Use `RouteControllerDidPassSpokenInstructionPoint` directly instead of converting to a raw value and back to an `NSNotification.Name` (×4).